File: Csharp/Laba6/Laba6/TestFolder/PalindromTest.cs
using NUnit.Framework;
using System;

namespace Laba6
{
	[TestFixture ()]
	public class PalindromTest
	{
		[Test ()]
		public void PalindromTester ()
		{
			int p = 30203;
			Assert.AreEqual (true, Palindrom.GetPalindrome (p));
		}
	}
}

File: Netbeans/Java/Lab6/test/model/PalindromTest.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import org.junit.Test;
import static org.junit.Assert.*;

/**
 *
 * @author prostrmk
 */
public class PalindromTest {
    
    public PalindromTest() {
    }

    @Test
    public void PalindromTest() {
        int a = 3113;
        assertEquals(true, Palindrom.getPalindrome(a));
    }
    
}
